en/docs/security-announcements/security-advisories/2016/WSO2-2016-0170.md

```diff
@@ -26,7 +26,7 @@ Apache Axis2 1.6.2 uses commons-httpclient-3.1.0 and host name verification shou
 
 
 ### IMPACT
-This allows man-in-the-middle attackers to spoof SSL servers via an arbitrary valid certificate.
+This allows Eavesdropper attackers to spoof SSL servers via an arbitrary valid certificate.
```

Comment:
Let's keep the term *man-in-the-middle* here.

Reply:
Thanks all. Will use `Eavesdropper`.